app/models/concerns/conference_override.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module ConferenceOverride
+  extend ActiveSupport::Concern
+
+  included do
+    def self.filtered_by_hashtag(hashtag)
+      where("CONCAT(' ', hashtag, ' ') ILIKE CONCAT('% ', ?, ' %') OR CONCAT(' ', hashtag, ' ') ILIKE CONCAT('% #', ?, ' %')", hashtag, hashtag)
```

Comment:
```suggestion
      where("(CONCAT(' ', hashtag, ' ') ILIKE ?) OR (CONCAT(' ', hashtag, ' ') ILIKE ?)", "% #{hashtag} %", "% ##{hashtag} %")
```
Is better to leave the interpolation to ruby